test/intl402/NumberFormat/test-option-roundingPriority-mixed-options.js

```diff
@@ -0,0 +1,94 @@
+    'maxSd-smaller-minFd' : { '1': '1', '1.500': '1.5', '1.625': '1.6', '1.750': '1.8', '1.875': '1.9', '2.000': '2' },
+    'maxSd-smaller-maxFd':  { '1': '1', '1.500': '1.5', '1.625': '1.6', '1.750': '1.8', '1.875': '1.9', '2.000': '2' },
+    'maxSd-larger-maxFd' : { '1': '1', '1.500': '1.5', '1.625': '1.6', '1.750': '1.8', '1.875': '1.9', '2.000': '2' },
+    'minSd-maxSd-smaller-minFd-maxFd' :{ '1': '1', '1.500': '1.5', '1.625': '1.6', '1.750': '1.8', '1.875': '1.9', '2.000': '2' },
+    'minSd-maxSd-larger-minFd-maxFd' :{ '1': '1.0', '1.500': '1.5', '1.625': '1.63', '1.750': '1.75', '1.875': '1.88', '2.000': '2.0' },
+    'same-minimums-maxFd-larger-maxSd': { '1': '1', '1.500': '1.5', '1.625': '1.6', '1.750': '1.8', '1.875': '1.9', '2.000': '2' },
+    'same-minimums-maxFd-smaller-maxSd': { '1': '1.0', '1.500': '1.5', '1.625': '1.63', '1.750': '1.75', '1.875': '1.88', '2.000': '2.0' }
+  },
+  'auto' : {
+      'same-minimums': { '1': '1.0', '1.500': '1.5', '1.625': '1.625', '1.750': '1.75', '1.875': '1.875', '2.000': '2.0' },
+      'same-maximums' : { '1': '1', '1.500': '1.50', '1.625': '1.6', '1.750': '1.8', '1.875': '1.9', '2.000': '2' },
+      'minSd-larger-minFd' : { '1': '1.00', '1.500': '1.50', '1.625': '1.625', '1.750': '1.75', '1.875': '1.875', '2.000': '2.00' },
+      'minSd-smaller-minFd' : { '1': '1', '1.500': '1.5', '1.625': '1.625', '1.750': '1.75', '1.875': '1.875', '2.000': '2' },
+      'minSd-smaller-maxFd' : { '1': '1', '1.500': '1.5', '1.625': '1.625', '1.750': '1.75', '1.875': '1.875', '2.000': '2' },
+      'minSd-larger-maxFd': { '1': '1.00', '1.500': '1.50', '1.625': '1.625', '1.750': '1.75', '1.875': '1.875', '2.000': '2.00' },
+      'maxSd-larger-minFd' : { '1': '1', '1.500': '1.5', '1.625': '1.63', '1.750': '1.75', '1.875': '1.88', '2.000': '2' },
+      'maxSd-smaller-minFd' : { '1': '1', '1.500': '1.5', '1.625': '1.6', '1.750': '1.8', '1.875': '1.9', '2.000': '2' },
+      'maxSd-smaller-maxFd':  { '1': '1', '1.500': '1.5', '1.625': '1.6', '1.750': '1.8', '1.875': '1.9', '2.000': '2' },
+      'maxSd-larger-maxFd' : { '1': '1', '1.500': '1.5', '1.625': '1.63', '1.750': '1.75', '1.875': '1.88', '2.000': '2' },
+      'minSd-maxSd-smaller-minFd-maxFd' :{ '1': '1', '1.500': '1.5', '1.625': '1.6', '1.750': '1.8', '1.875': '1.9', '2.000': '2' },
+      'minSd-maxSd-larger-minFd-maxFd' :{ '1': '1.00', '1.500': '1.50', '1.625': '1.625', '1.750': '1.75', '1.875': '1.875', '2.000': '2.00' },
+      'same-minimums-maxFd-larger-maxSd': { '1': '1', '1.500': '1.5', '1.625': '1.6', '1.750': '1.8', '1.875': '1.9', '2.000': '2' },
+      'same-minimums-maxFd-smaller-maxSd': { '1': '1', '1.500': '1.5', '1.625': '1.625', '1.750': '1.75', '1.875': '1.875', '2.000': '2' }
+  }
+}
+
+const optionsMatrix = {
+  'same-minimums' :{ minimumSignificantDigits: 2, minimumFractionDigits: 2},
+  'same-maximums' :{ maximumSignificantDigits: 2, maximumFractionDigits: 2},
```

Comment:
```suggestion
  'same-maximums': { maximumSignificantDigits: 2, maximumFractionDigits: 2},
```
can the colons and braces be consistently spaced throughout?